lib/generators/minitest/install/templates/test/test_helper.rb.tt

```diff
@@ -4,7 +4,7 @@ require "rails/test_help"
 require "minitest/rails"
 
 # Consider setting MT_NO_EXPECTATIONS to not add expectations to Object.
-# ENV["MT_NO_EXPECTATIONS"] = true
+# ENV["MT_NO_EXPECTATIONS"] = 'true'
```

Comment:
```suggestion
# ENV["MT_NO_EXPECTATIONS"] = "true"
```

Reply:
Oh, I see, this project use double quotes in `.rubocop.yml`
https://github.com/minitest/minitest-rails/blob/80947c0409b6a5d69e62b1c233d64aa9e5ecf398/.rubocop.yml#L73

I'll fix it now